Resources/ArmTemplates/template.json

```diff
@@ -0,0 +1,350 @@
+        "appInsightsName": "[concat(parameters('projectPrefix'), '-occskills-ai')]"
+      },
+      {
+        "functionType": "GetAlternativeCareersAndSkillsForOccupation",
+        "functionName": "[concat(parameters('projectPrefix'), '-altcar-fa')]",
+        "appInsightsName": "[concat(parameters('projectPrefix'), '-altcar-ai')]"
+      },
+      {
+        "functionType": "GetSkillsGapForOccupations",
+        "functionName": "[concat(parameters('projectPrefix'), '-skillsgap-fa')]",
+        "appInsightsName": "[concat(parameters('projectPrefix'), '-skillsgap-ai')]"
+      }
+    ]
+  },
+  "resources": [
+    {
+      "apiVersion": "2017-05-10",
+      "name": "ServiceTaxonomyStorage",
+      "type": "Microsoft.Resources/deployments",
+      "properties": {
+        "mode": "Incremental",
+        "templateLink": {
+          "uri": "[concat(variables('BuildingBlocksDfcBaseUrl'),'storage-account.json')]",
+          "contentVersion": "1.0.0.0"
+        },
+        "parameters": {
+          "storageAccountName": {
+            "value": "[parameters('storageAccountName')]"
+          }
+        }
+      }
+    },
+    {
+        "name": "servicetaxonomy-apim-logger",
+        "type": "Microsoft.Resources/deployments",
+        "resourceGroup": "[parameters('ApimResourceGroup')]",
+        "apiVersion": "2017-05-10",
+        "properties": {
+            "mode": "Incremental",
+            "templateLink": {
+                "uri": "[concat(variables('BuildingBlocksDfcBaseUrl'), 'APIM/apim-logger.json')]",
+                "contentVersion": "1.0.0.0"
+            },
+            "parameters": {
+                "apimServiceName": {
+                    "value": "[parameters('ApimServiceName')]"
+                },
+                "productDisplayName": {
+                    "value": "[variables('ApimProductName')]"
+                }
+            }
+        }
+    },
+    {
+        "name": "service-taxonomy-apim-product",
+        "type": "Microsoft.Resources/deployments",
+        "resourceGroup": "[parameters('ApimResourceGroup')]",
+        "apiVersion": "2017-05-10",
+        "dependsOn": [
+          "servicetaxonomy-apim-logger"
+        ],
+        "properties": {
+            "mode": "Incremental",
+            "templateLink": {
+                "uri": "[concat(variables('BuildingBlocksDfcBaseUrl'), 'APIM/apim-product.json')]",
+                "contentVersion": "1.0.0.0"
+            },
+            "parameters": {
+                "allowAnonymousAccess": {
+                    "value": false
+                },
+                "apimServiceName": {
+                    "value": "[parameters('ApimServiceName')]"
+                },
+                "productDisplayName": {
+                    "value": "[variables('ApimProductName')]"
+                }
+            }
+        }
+    },
+    {
+      "apiVersion": "2017-05-10",
+      "name": "[concat('service-taxononmy-', variables('allFunctions')[copyIndex()].functionName)]",
```

Comment:
Only an issue for log readability but should this be functionType as that's the name of the API this resource is deploying?

Reply:
updated to be functionType to improve clarity